ambry-replication/src/main/java/com/github/ambry/replication/ReplicaThread.java

```diff
@@ -198,6 +204,13 @@ public ReplicaThread(String threadName, FindTokenHelper findTokenHelper, Cluster
     }
     this.maxReplicaCountPerRequest = replicationConfig.replicationMaxPartitionCountPerRequest;
     this.leaderBasedReplicationAdmin = leaderBasedReplicationAdmin;
+    try {
+      fileManager = Utils.getObj(replicationConfig.backupCheckFileManagerType, replicationConfig, metricRegistry);
```

Comment:
Again backup checker specific code here.

Reply:
The default backup checker filemgr is empty